app/src/main/kotlin/com/github/oryanmat/trellowidget/util/network/DataStatus.kt

```diff
@@ -0,0 +1,13 @@
+package com.github.oryanmat.trellowidget.util.network
+
+data class DataStatus<out T>(val status: Status, val data: T? = null, val msg: String? = null) {
```

Comment:
This name is meh. This is more of a `ResponseEntity` or just `Response`. 

Also consider making it a sealed interface with 2 implementations. That would make expressing the difference between success and failure cleaner.

Reply:
> This name is meh. This is more of a `ResponseEntity` or just `Response`.

I would suggest `ApiResponse`.

> Also consider making it a sealed interface with 2 implementations.

Good point. I will do the implementation soon.